pkg/exchange/bitget/types.go

```diff
@@ -425,3 +427,63 @@ type AccountEvent struct {
 	actionType ActionType
 	instId     string
```

Comment:
why not make them public? since it's bitget's type, I think it's safe?

Reply:
I don't think the outer layer will use this variable because we will convert it into a balance event and emit it.

Reply:
Of course, I can also make it public, no problem.